bindings_ffi/Makefile

```diff
@@ -35,39 +37,28 @@ aarch64-apple-ios:
 	cargo build --target $@ --target-dir ./target --release
 
 $(LIB): $(ARCHS_IOS) $(ARCHS_MAC) aarch64-apple-ios
-	rm -rf lipo_macos lipo_ios_simulator
-	mkdir -p lipo_macos lipo_ios_simulator
-	lipo -create -output lipo_ios_simulator/libxmtp_rust_swift.a $(foreach arch,$(ARCHS_IOS),$(wildcard target/$(arch)/release/$(LIB)))
-	lipo -create -output lipo_macos/libxmtp_rust_swift.a $(foreach arch,$(ARCHS_MAC),$(wildcard target/$(arch)/release/$(LIB)))
-
-move_gen:
-	rm -rf Generated
-	# Clean and build to regenerate swift-bridge "Generated" directory
-	cargo clean
-	cargo build
-	rm -rf include/Generated
-	# Move "Generated" directory to "include"
-	cp -r Generated include/
-	# Delete the *.swift files within include/Generated but
-	# Keep top-level Generated for push-to-swift step where .swift files are
-	# copied over to another repo (xmtp-rust-swift)
-	find include/Generated -type f -name "*.swift" -delete
 
-framework: move_gen $(LIB)
-	rm -rf XMTPRustSwift.xcframework
+framework: swift $(LIB)
```

Comment:
This isn't really used during the Build action because the matrix compilation produces different structures, but I used this to test the process locally.